externals/llvm-external-projects/torch-mlir-dialects/include/torch-mlir-dialects/Dialect/Tcp/IR/TcpOps.td

```diff
@@ -320,4 +320,25 @@ def Tcp_SqrtOp : Tcp_UnaryElementwiseOp<"sqrt", [SameOperandsAndResultElementTyp
   let assemblyFormat = "$in attr-dict `:` type($in) `->` type($out)";
 }
 
+def Tcp_ConcatOp : Tcp_Op<"concat", [SameOperandsAndResultElementType]> {
+  let summary = "Concatenation of input tensors";
+
+  let description = [{
+    Concatenates tensors of same datatype and rank into a single tensor on specific single axis.
```

Comment:
Maybe also add the precondition that the dims on the other axes must match.  (Undefined behavior if they don't match.)

Reply:
But `::verify` will check this only if the shapes are static.  My point is that we should document that the op has undefined behavior if the dims are dynamic (`?`) and don't match at runtime.

Reply:
Yes and you're right that it isn't documented for these other binary ops either.  I think it should be documented in `Tcp_BinaryElementwiseOp`.

So feel free to skip this comment for now, I'll try fix this documentation issue in a later PR.